src/components/NewAccountPage/NewAccountPage.jsx

```diff
@@ -151,18 +196,24 @@ export default () => {
                <StyledField
                   required
                   type="text"
-                  placeholder="Password"
+                  // placeholder="Password"
                   defaultValue={password}
                   onChange={(e) => setPassword(e.target.value)}
                />
+               <StyledHideButton>
+                  Hide <FontAwesomeIcon icon={faEye} />
+               </StyledHideButton>
```

Comment:
Make sure this button is always at the right end of the password input field. This can be done using flex-end from flexbox.